src/test/test.js

```diff
@@ -42,47 +42,68 @@ describe('routes : index', () => {
 
 describe('routes : question', () => {
   describe('GET /api/v1/questions', () => {
-    it('should respond a success message along with all questions', (done) => {
+    it('should respond a success message and all questions', (done) => {
       chai.request(server)
         .get('/api/v1/questions')
         .end((err, res) => {
           res.status.should.equal(200);
           res.type.should.equal('application/json');
-          res.body.status.should.eql('success');
+          res.body.status.should.eql('Success');
+          res.body.message.should.eql('Returning list of questions');
+          res.body.data.should.include.keys(
+            'count', 'questions'
+          );
+          res.body.data.count.should.eql(2);
           done();
         });
     });
   });
 
   describe('GET /api/v1/questions/:questionId', () => {
-    it('should respond with a success message along with a single question', (done) => {
+    it('should respond with a success message and a single question', (done) => {
       chai.request(server)
         .get('/api/v1/questions/1')
         .end((err, res) => {
           res.status.should.equal(200);
           res.type.should.equal('application/json');
-          res.body.status.should.eql('success');
+          res.body.status.should.eql('Success');
+          res.body.message.should.eql('Returning question');
           res.body.data.should.include.keys(
-            'id', 'title', 'questionBody'
+            'id', 'title', 'questionBody', 'answers'
           );
+          res.body.data.title.should.equal('Qui aggredior inveniant desumptas aliquibus sic medicinam nam?');
+          res.body.data.questionBody.should.equal('Ipsius cupere vulgus tes hos. Eidem motus vos lucis ibi res mundo. Sit scribere quicquam ibi imponere. Ab generis re de se essendi nunquam. Generis vigilia futurus quodque co calebat spatium id.');
```

Comment:
This line is too long. You are expected to use Airbnb style guide which makes use of 100 max length, but the max length should be set to 80. In essence you should not have a line longer than 80 characters. Add that your eslint configuration so it will notify you when any line goes beyond 80.

Reply:
Make the necessary changes in the rest of your files.